go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/dell/csm-replication
 
-go 1.22
-
-toolchain go1.22.0
```

Comment:
Hi @shaynafinocchiaro 
I see that the Bot has updated the toolchain version in driver repo PRs, and here it is removing it.

Reply:
Good catch. I'll work on a fix for this. For now, I'll fix it manually.